libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/StylingEditorDialogFooter.tsx

```diff
@@ -0,0 +1,53 @@
+// (C) 2022 GoodData Corporation
+
+import { Button } from "../../Button";
+import React from "react";
+import { useIntl } from "react-intl";
+
+/**
+ * @internal
+ */
+export interface IStylingEditorDialogFooterProps {
+    link: {
+        text: string;
+        url: string;
+    };
+    disableSubmit?: boolean;
+    onCancel?: (data?: any) => void;
```

Comment:
pls try to remove any type. onCancel probably does not require any param and onSubmit should provide whole StylingPickerItem, right?